helpers/path.ts

```diff
@@ -72,11 +72,31 @@ export function collapsedDirFromPath(path: string | null): string | null {
   }
 }
 
+/**
+ * Return dir for a given file path
+ * * @example
+ * // returns 'src/main/js/components/navigator/app/models'
+ * dirFromPath('src/main/js/components/navigator/app/models/state.js')
+ */
+export function dirFromPath(path: string | null): string | null {
+  if (typeof path === 'string') {
+    if (path === '/') {
+      return path;
+    } else {
+      const tokens = path.split('/');
+      tokens.pop();
+      return tokens.join('/');
```

Comment:
How about `path.substring(0, path.lastIndexOf('/'))` ?

Reply:
Yeah I know it's a closed PR, just curious ^^'

Reply:
That would have indeed been much more elegant 🙂 .